packages/remirror__cli/src/utils/build-package.ts

```diff
@@ -83,7 +83,19 @@ export async function buildPackage(
                 // We need to set the ESBuild target to esnext so that it can
                 // pass through the decorators syntax. So we need another layer
                 // to do the transpilation. This is inefficient but it works.
-                presets: [['@babel/preset-env', { targets: { node: 14 }, modules: false }]],
+                presets: [
+                  [
+                    '@babel/preset-env',
+                    {
+                      targets: {
+                        node: '14',
+                        // See https://remirror.io/docs/advanced/browser-support
+                        browsers: 'since 2017',
```

Comment:
@ocavue This hasn't been changed in a while, perhaps we could update this to `since 2019` or something?

Reply:
I don't have a s strong opinion of it. I think it's ok if you didn't see any issues with your project.